### ISSUE TITLE
Warn: TLS is not enabled, See URL is 404

### ISSUE THREAD

**Issue by woohgit** [jenkins-x/jx#6192]
### Summary 

When running `jx boot` and the TLS is not enabled, I get a warn message but the See URL is 404.

```
WARNING: TLS is not enabled so your webhooks will be called using HTTP. This means your webhook secret will be sent to your cluster in the clear. See https://jenkins-x.io/architecture/tls for more information
```

### Steps to reproduce the behavior

Run `jx boot` with a default settings and click on the link.

### Expected behavior

The link shouldn't be 404.

### Actual behavior

It's 404.

### Jx version

The output of `jx version` is:

```
NAME               VERSION
jx                 2.0.1011
Kubernetes cluster v1.13.11-gke.14
kubectl            v1.15.2
git                2.24.0
Operating System   "Manjaro Linux"

```

### Jenkins type

- [x ] Serverless Jenkins X Pipelines (Tekton + Prow)
- [ ] Classic Jenkins

### Kubernetes cluster

GKE

### Operating system / Environment

```

 ██████████████████  ████████     wooh@carbonx
 ██████████████████  ████████     OS: Manjaro 18.1.3 Juhraya
 ██████████████████  ████████     Kernel: x86_64 Linux 5.3.12-1-MANJARO
 ██████████████████  ████████     Uptime: 1d 2h 58m
 ████████            ████████     Packages: 1312
 ████████  ████████  ████████     Shell: bash 5.0.11
 ████████  ████████  ████████     Resolution: 1920x1080
 ████████  ████████  ████████     DE: Xfce4
 ████████  ████████  ████████     WM: Xfwm4
 ████████  ████████  ████████     WM Theme: Adapta-Maia
 ████████  ████████  ████████     GTK Theme: Adapta-Eta-Maia [GTK2]
 ████████  ████████  ████████     Icon Theme: Papirus-Maia
 ████████  ████████  ████████     Font: Noto Sans 10
 ████████  ████████  ████████     Disk: 136G / 225G (64%)
                                  CPU: Intel Core i7-8550U @ 8x 4GHz [53.0°C]
                                  GPU: Mesa DRI Intel(R) UHD Graphics 620 (Kabylake GT2) 
                                  RAM: 8735MiB / 15778MiB

```

**Comment by abayer** [comment]
The URL was fixed in jx with #6190, but I'll add a redirect as well.

**Comment by abayer** [comment]
PR to add the redirect up at https://github.com/jenkins-x/jx-docs/pull/2250 - but it's...not ideal, since I can't find a way to make Hugo redirect an old page to an anchor on a new page.
